ibis/backends/bigquery/__init__.py

```diff
@@ -231,6 +231,45 @@ def project_id(self):
     def dataset_id(self):
         return self.dataset
 
+    def create_schema(
+        self,
+        name: str,
+        database: str | None = None,
+        force: bool = False,
+        collate: str | None = None,
+        **options: Any,
+    ) -> None:
+        code = "CREATE SCHEMA"
```

Comment:
Not a hill anyone has to die on, but what do you think about calling this (and the related) variables `create_stmt` and `drop_stmt`, respectively?

Reply:
Done!